lib/src/query_service.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:js';
+
+import 'gapi.dart';
+import 'json_js.dart';
+import 'insertion_order_parser.dart';
+import 'proto/insertion_order.pb.dart';
+
+class QueryService {
+  QueryService._private();
+
+  static final QueryService _singleton = QueryService._private();
+
+  factory QueryService() {
```

Comment:
prefer using @Injectable rather than using singleton pattern
same in a couple other places.

Reply:
Singleton has been removed for this class, and will do the same for other places in a separate PR (new issue #32)